tests/pkl_codegen_java/TestPklCodegen.java

```diff
@@ -0,0 +1,40 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+import org.pkl.config.java.ConfigEvaluator;
+import org.pkl.config.java.Config;
+import org.pkl.config.java.JavaType;
+import org.pkl.core.ModuleSource;
+
+import java.util.List;
+
+import org.Dogs;
+
+
+public class TestPklCodegen {
+    @Test
+    public void verifyReadingPklFiles() {
+        ConfigEvaluator evaluator = ConfigEvaluator.preconfigured();
+        Config config = evaluator.evaluate(ModuleSource.modulePath("srcs/litter.pkl"));
+
+        org.Dogs.Litter actual = config.get("puppies").as(JavaType.of(Dogs.Litter.class));
+        org.Dogs.Litter expected = new org.Dogs.Litter(List.of(new Dogs.Dog("Bruno"), new Dogs.Dog("Bruce")));
+
+
```

Comment:
We may want to integrate one of the formatter rule sets to take of stuff like this and anything else that may have been missed with these Java files.

Reply:
I've removed this one.